sycl/test/devicelib/assert-windows.cpp

```diff
@@ -0,0 +1,68 @@
+// REQUIRES: cpu,windows
+// RUN: %clangxx -fsycl -c %s -o %t.o
+// RUN: %clangxx -fsycl %t.o %llvm_build_libs_dir/../bin/libsycl-msvc.o -o %t.out
+//
+// MSVC implementation of assert does not call an unreachable built-in, so the
+// program doesn't terminate when fallback is used.
+//
+// FIXME: SPIR-V Unreachable should be called from the fallback
+// explicitly. Since the test is going to crash, we'll have to follow a similar
+// approach as on Linux - call the test in a subprocess.
+//
+// RUN: env SYCL_PI_TRACE=1 SYCL_DEVICELIB_LINK_FALLBACK=1 CL_CONFIG_USE_VECTORIZER=False SYCL_DEVICE_TYPE=CPU %t.out >%t.stdout.fallback 2>%t.stderr.fallback
```

Comment:
`SYCL_DEVICELIB_LINK_FALLBACK` - I cannot find any code that handles this env variable in the patch